cert.go

```diff
@@ -70,13 +71,15 @@ func (m *mkcert) makeCert(hosts []string) {
 			tpl.IPAddresses = append(tpl.IPAddresses, ip)
 		} else if email, err := mail.ParseAddress(h); err == nil && email.Address == h {
 			tpl.EmailAddresses = append(tpl.EmailAddresses, h)
+		} else if uriName, err := url.Parse(h); err == nil && uriName.Scheme != "" {
```

Comment:
I'm a bit worried this will catch things some people want as hostnames. Anything with a `:` will parse as an URI with `Scheme != ""`, and sometimes people use `:` in hostnames, for example.

Are there other filters we can apply while still working for your use case?

Reply:
For my concrete use case I want Spiffe URIs, so checking for value URLs plus the prefix `spiffe://` would be fine too (for me)

Reply:
I'll update the PR to require the scheme `spiffe`.

Reply:
No need to make it Spiffe specific. Looks like those are URLs with the full `://`, so we can just check `Scheme != "" && Host != ""` (because without `://` only Scheme and Opaque get populated).

Reply:
Thanks! Just updated the PR with this check.